openselery/openselery.py

```diff
@@ -261,13 +278,18 @@ def choose(self, contributors, repo_path, weights):
             self.log("  > via project '%s'" % contributor.fromProject)
         return recipients
 
-    def visualize(self):
-      try:
-        visualizeTransactions(self.config.result_dir)
-      except Exception as e:
-        print("Error creating visualization", e)
+    def visualize(self, receiptFilePath, transactionFilePath):
+        if transactionFilePath:
+            self.log("Creating visualizations for past transactions [%s]" % transactionFilePath)
```

Comment:
The `receiptFilePath` argument is unused, we can remove it.

Reply:
it is not used right now, but it does show the clear intend (see the main script api) to visualize more then just past transactions.

You kind of HAVE to do something with the receipt, either by exporting it to pdf or doing something fancy with it.

My comment above the "visualize" function would be 
``` ### create exports of given inputs which can be used to generate further transaction/money based data ```
with taht kind of logic in mind, the receipt should be exported to something useful as soon as possible and is therefore needed as an argument here.